src/screens/CreateWalletScreen.tsx

```diff
@@ -112,22 +120,31 @@ export class CreateWalletScreen extends React.PureComponent<Props, State> {
           email,
           flowType: ConfirmAddressFlowType.SUBSCRIBE,
           walletsToSubscribe: [wallet],
-          onSuccess: this.navigateToSuccesfullNotificationSubscriptionMessage,
+          onSuccess: isAfterAirdrop()
+            ? this.navigateToSuccesfullNotificationSubscriptionMessage
+            : () => this.navigateToAirdropWalletSubscription(wallet, true),
         }),
-      onBack: () => this.props.navigation.navigate(Route.MainTabStackNavigator, { screen: Route.Dashboard }),
+      onBack: () =>
+        isAfterAirdrop()
+          ? this.props.navigation.navigate(Route.MainTabStackNavigator, { screen: Route.Dashboard })
+          : this.navigateToAirdropWalletSubscription(wallet, true),
```

Comment:
this.navigateToAirdropWalletSubscription(wallet)

Reply:
a shit, true, thanks

Reply:
done